5_databases/week3/homework3

```diff
@@ -0,0 +1,140 @@
+-- 2.Add a new reservation
+INSERT INTO reservation (number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email)VALUES (5, 2, CURDATE(), '22334455', 'Sam Jones', 'samjones@gmail.com');
+-- 3.Get a reservation with any id, fx 1
+SELECT *
+FROM reservation
+WHERE id = 1;
+-- 4.Update a reservation with any id, fx 1. Update any attribute fx the number_of_guests or multiple attributes
+UPDATE reservation
+SET number_of_guests = 10, meal_id = 1, created_date = CURDATE(), contact_phonenumber = '22334455', contact_name = 'Sam Jones', contact_email = 'samjones@gmail.com'
+-- 5.Delete a reservation with any id, fx 1
+DELETE FROM reservation
+WHERE id = 1;
+-- Review Queries
+-- 1.Get all reviews
+SELECT *
+FROM Review;
+-- 2.Add a new review
+INSERT INTO Review(title, description, meal_id, stars, created_date)
+VALUES('Excellent', 'Perfect meal', 1, 5, CURDATE());
+-- 3.Get a review with any id, fx 1
+SELECT *
+FROM Review
+WHERE id = 1;
+-- 4.Update a review with any id, fx 1. Update any attribute fx the title or multiple attributes
+UPDATE Review
+SET title = 'update title', description = 'update description', meal_id = 1, stars = 5, created_date = CURDATE();
+-- 5.Delete a review with any id, fx 1
+DELETE FROM Review
+WHERE id = 1;
+-- Additional queries
+-- Now add a couple of different meals, reservations and reviews with different attributes. With those meals create the following queries
+INSERT INTO meal (title, description, location, when_date, max_reservations, price, created_date)
+VALUES ('FUFU', 'Delicious Starchy', 'Ghana', '2023-11-21', 10, 50.00, CURDATE()),
+('Fante Fante', 'Spicey fish stew with Kenkey(made of corn)', 'Cape Coast', '2023-12-31 15:41:20', 10, 40.00, CURDATE()),
+('Rød grød med fløde', 'Delicious Danish dessert', 'Copenhagen', '2023-11-22 18:00:00', 10, 80.00, CURDATE()),
+('Pasta Carbonara', 'Classic Italian pasta dish', 'Rome', '2023-11-23 19:30:00', 10, 60.00, CURDATE()),
+('Indonesian Meal', 'A delicious traditional Indonesian meal', 'Indonesia', '2023-12-12 13:10:00', 10, 90.00, CURDATE()),
+('Sushi Night', 'Fresh Japanese sushi', 'Tokyo', '2023-11-25 20:00:00', 12, 150.00, CURDATE()),
+('Burger', 'American burger', 'New York', '2023-11-28 19:11:10', 10, 150.00, CURDATE()),
+('Tacos', 'Mexican tacos', 'Mexico', '2023-12-01 10:00:00', 10, 110.00, CURDATE());
+INSERT INTO reservation (number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email)
+VALUES (5, 1, CURDATE(), '22334455', 'Sam Jones', 'samjones@gmail.com'),
+(6,2, CURDATE(), '32334455', 'Peter Tosh', petertosh@gmail.com '),
+(8,4, 2, CURDATE(), '9233455', 'Bob Marley', 'bobmarley@gmail.com'),
+(10,3, CURDATE(), '112334455', 'Sam james', '
+SELECT *
+FROM meal
+-- Functionality
+-- 1.Get meals that has a price smaller than a specific price fx 90
+SELECT *
+FROM meal
+WHERE price < 90;
+-- 2.Get meals that still has available reservations
+SELECT *
+FROM meal
+WHERE max_reservations > 0;
```

Comment:
Here you only get the reservations with a max_reservations higher than 0. You need to take into account how many reservations have been created for this meal and how many people each reservation is for. How can you modify the below query to do that?
```sql

SELECT meal.id, meal.title, meal.max_reservations, SUM(reservation.???) AS reservations
FROM meal
LEFT JOIN reservation ON meal.id = reservation.meal_id
GROUP BY meal.id
HAVING meal.max_reservations > ???;
```